changelog/1.28.4.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "1.28.4"
+description: "Released on 03/17/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.28.4.
+
+### Features ✨
+
+- web: added ability to host the **Open in Coder** SVG image in Coder in
```

Comment:
@vapurrmaid wdyt?

Reply:
We now use the SVG hosted in Coder by default, so we might want to capture a few things here:

- a) this is a fix for firewalled environments that can't access an external cdn
- b) this is the new default, but our CDN will continue to be supported with no deprecation timeline as of yet.

I'm not sure how to word it, but here's a rough idea:


```
web: the **Open in Coder** SVGs are now statically hosted in Coder and will render properly in firewalled networks. Our external CDN will continue to be supported, but we default to pulling the SVG from Coder.
```

(feat --> fix)

Reply:
Not sure if the first use of static is necessary, but what about

```
web: added [static] hosting for the **Open in Coder** SVGs in Coder so that the icons render properly in firewalled networks. Coder will continue to support the existing external CDN for such icons, but the default will be the SVG hosted by Coder.
```

Reply:
I think either/or works, and since brevity is helpful in a changelog, we can do it without `static`. This is great!!!

Always love your way with words @khorne3 :taco: :tada: !